strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -102,24 +118,36 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         Returns:
             float: measured value
         """
-        eps = kwargs.get('eps', 0.0002)
+        if select is not None and shots != 1:
+            raise ValueError("Post selection and multiple shots is not implemented")
+        eps = kwargs.get("eps", 0.0002)
```

Comment:
Do we document this tolerance anywhere?

Reply:
@nquesada any reason for this specific tolerance?

Reply:
No, other that it seemed small enough at the time.